config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
   root 'users#index'
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+  resources :users, only:  [:index, :show,  ]  do
+    resources :posts, only: [:index, :show,:new, :create] do
+      resources :comments, only: [:create]
```

Comment:
Hello @enaburekhan, with the way my code is currently structured a `new` method doesn't serve any purpose. It was left out deliberately because of this reason.

Reply:
This change has been done well